src/AoTStage1/AoTStage1.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.0-alpha.1.22608.3" />
```

Comment:
If this is a package that is shipped alongside with aspnet and it has the SAME version, then there is a variable that will contain it: `$(MicrosoftAspNetCoreAppPackageVersion)`